src/sentry/shared_integrations/client/base.py

```diff
@@ -464,3 +468,29 @@ def disable_integration(self, buffer) -> None:
                 data={"provider": rpc_integration.provider},
             )
         return
+
+    def disable_plugin(self, buffer) -> None:
+        plugin = plugins.get(self.slug)
```

Comment:
Do we actually have a `self.slug`?

Reply:
Yeah we don't I changed back to plugin_name, I was looking at the plugin instead of the client